Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM praekeltfoundation/django-bootstrap:py3.6-jessie
-RUN apt-get-install.sh git \
-    redis-server supervisor libpq-dev gcc && \
-    ln -s /usr/bin/nodejs /usr/bin/node
+FROM praekeltfoundation/django-bootstrap:latest
 
 #Upgrade Nodejs to v4.9 as the current version v4.6 doesn't appear to have support for object.assign 
 #Check https://node.green for compatibility
-RUN apt-get update && apt-get install curl -y && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install nodejs -y
+RUN apt-get update && apt-get install git redis-server supervisor libpq-dev gcc curl -y && curl -sL https://deb.nodesource.com/setup_4.x | bash && apt-get install nodejs -y && apt-get autoremove -y && apt-get clean
```

Comment:
I think we should probably be using the `apt-get-install.sh` and `apt-get-purge.sh` scripts from the base python image to leverage all that it gives us, but that's unrelated to the changes you've made